README.md

```diff
@@ -1,73 +1,30 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Awesome books: plain JavaScript with objects
 
+> Basic website that allows users to add/remove books from a list using Javascript.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+- HTML
+- CSS
+- Javascript
 
 ## Authors
 
-👤 **Author1**
+👤 **Romina Patiño**
+
+- GitHub: [@rominana](https://github.com/rominana)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Rodrigo Johann Reckziegel Nunes**
 
-👤 **Author2**
+- GitHub: [@hannziegel](https://github.com/hannziegel)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 ## 🤝 Contributing
```

Comment:
* [OPTIONAL] I strongly suggest that update your readme file with the setup instructions of how a visitor can successfully install and run your app 🙏